aws/v15.0.0/release.diff

```diff
@@ -0,0 +1,71 @@
+                                                                >  # Generated with:
+                                                                >  # devctl release create --name 15.0.0 --provider aws --base 14.
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v14.2.2                                                 |    name: v15.0.0
+spec:                                                              spec:
+  apps:                                                              apps:
+                                                                >    - componentVersion: 1.1.0
+                                                                >      name: aws-ebs-csi-driver
+                                                                >      version: 2.1.0
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.6.1                                                     version: 1.6.1
+  - componentVersion: 1.3.1                                          - componentVersion: 1.3.1
+    name: cert-manager                                                 name: cert-manager
+    version: 2.7.1                                              |      version: 2.7.0
```

Comment:
I recently updated the realease.diff to compare this release with v14.2.2 and I noticed that we will be downgrading cert-manager from v2.7.1 to v2.7.0. Is this intentional or should we bump cert-manager to v2.7.1?

Reply:
I upgraded cert-manager to 2.7.1. It only contains helm changes.